src/components/Login/Login.jsx

```diff
@@ -1,7 +1,94 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import './Login.scss';
 
 function Login() {
-  return <h1>Login</h1>;
+  const navigate = useNavigate();
+  const [inputValue, setInputValue] = useState({
+    username: '',
+    password: '',
+  });
+  const handleChangeInput = e => {
+    const { name, value } = e.target;
+    setInputValue({ ...inputValue, [name]: value });
+  };
+  const handleLogin = e => {
+    e.preventDefault();
+
+    fetch('http://192.168.14.221:8000/users/signin', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+      },
+      body: JSON.stringify({
+        username: inputValue.username,
+        password: inputValue.password,
+      }),
```

Comment:
```suggestion
      body: JSON.stringify(inputValue),
```
결국 state 값을 넣는 것이기 때문에 이런 방식으로 구현하는 것이 좀 더 효율적이겠네요!

Reply:
오.... 수정했습니다